modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,177 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-network-issues.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-manifest-file-multiple-interfaces-br-ex_{context}"]
+= Creating an additional OVN-Kubernetes `br-ex` bridge network
+
+You can create an additional `br-ex` bridge network that has a different configuration to the default `br-ex` bridge. You must define the additional bridge in the target interface definition of a machine config manifest file.  
+
+After you create and edit the manifest file, the Machine Config Operator completes tasks in the following order:
+
+* Drains nodes in singular order based on the selected machine config pool.
+* Injects Ignition configuration files into each node, so that each node receives the additional `br-ex` bridge network configuration.
+* Restarts the node with the modified override file.
+* Executes the `configure-ovs.sh` shell script that references the new interface definition.
+* Adds `br-ex` and `br-ex1` to the host node.
+* Uncordons the nodes. After all the nodes return to the `Ready` state, the Operator applies the `k8s.ovn.org/l3-gateway-config` annotation to each node.
+
+For more information about useful situations for the additional `br-ex` bridge and a situation that always requires the default `br-ex` bridge, see "Configuration for a localnet topology".
+
+.Procedure
+
+. Convert the rule to Base64 encoded by running the following command:
++
+[source,terminal]
+----
+$ base64 /path/to/file/
+----
+
+. Complete one of the following steps to define a secondary interface for your network. Ensure that the secondary interface is not bound to the `br-ex` bridge.
++
+[IMPORTANT]
+====
+Do not use the Kubernetes NMState Operator to define or a `NodeNetworkConfigurationPolicy` (NNCP) manifest file to define the secondary interface.
+====
++
+.. Define a secondary interface in a machine config manifest file:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 11-master-bond0
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;base64,<encoded_base64_string> <1>
+        path: /etc/NetworkManager/system-connections/bond0.nmconnection
+        filesystem: root
+        mode: 0600
+      - contents:
+# ...
+----
+<1> The Base64 encoded string that you have generated in the previous step.
++
+.. Create a connection profile for the bonded secondary interface and save the file in the `/etc/NetworkManager/system-connections/secondary-iface.nmconnection` path. For example, create the `bond1.nmconnection` file in your local directory with the following content:
++
+[source,ini]
+----
+[connection]
+id=eno2
+type=ethernet
+interface-name=eno2
+master=bond0
+slave-type=bond
+autoconnect=true
+autoconnect-priority=20
+
+[ethernet]
+mtu=9000
```

Comment:
I removed the mtu=<value> references in my big update above about these MC objects, as they can cause problems elsewhere with MTU migrations if hard-coded into the interface directly. better to omit.

Reply:
:heavy_check_mark: